github-management/org-owners-guide.md

```diff
@@ -76,6 +78,24 @@ for all orgs going forward.  Notable discrepancies at the moment:
   org. This was a mistake that swapped the usual convention, and we would like
   to rename the team
 
+### Structure and Process
+
+Guidelines on how to create and structure teams are described below:
+
+- Teams should not have any `maintainers` and everyone who should be part of
```

Comment:
Could we tweak the language here? Teams will still need to have maintainers if the org owners are part of the team, so this is not strictly accurate. Also, we should make it easy for people to discover an org owner who is part of that team, so maybe a link to the @kubernetes/owners team would be helpful.

Reply:
We've got the note mentioning that org admins still need to be in the `maintainers` list, I think that's clear enough. But it indeed might be good to link to `@kubernetes/owners` as well